### DISCUSSION TITLE
Need to access a dll for my .net application

### DISCUSSION THREAD

**Discussion by wes-kay** [ImageMagick/ImageMagick#2126] [Help]
Hello, I'm looking to add the .dll (Magick, and Magick convert) to my own application to fire off some simple CLI command line tasks on images (Can't install IM on users machines). 

A little confused on how to get them as stand alone. Do I need to install the `static_dll.exe` then copy the dlls from the project folders?

**Comment by dlemstra** [comment]
If you are creating a .NET application you could consider using the [https://github.com/dlemstra/Magick.NET](Magick.NET) library.

**Reply by wes-kay** [reply]
Thank you, yes I was looking at that, but just wanted to see if there was a faster way to include the libraries as a dll, and also since I only need those two tools. 
Just to confirm, your suggestions is my only option? I can't just grab the two tool libraries?

**Reply by dlemstra** [reply]
It probably depends on what you want to do but with my library it will probably be the easiests to put it on another person their computer.

**Reply by wes-kay** [reply]
I'm just looking for the magick, and magick convert dll. Is there such a thing?
